README.md

```diff
@@ -46,9 +45,33 @@ Run local Data Explorer with the 1000 Genomes dataset:
 
 ## Architecture overview
 
-For local development, an nginx reverse proxy is used to get around CORS.
+The basic flow:
 
-![Architecture overview](https://i.imgur.com/IZLbPx9.png)
+1. Index dataset into Elasticsearch using an indexer from https://github.com/DataBiosphere/data-explorer-indexers
+2. Run the servers in this repo to display Data Explorer UI
+
+GCP deployment:
+
+![GCP deployment architecture](https://i.imgur.com/mCjDv47.png)
```

Comment:
This diagram doesn't seem right, I think you may have just mixed up the order? It seems to me like API server should be in the middle (it talks to elasticsearch, not the UI server as shown).

Reply:
Done